corehq/apps/es/tests/test_client.py

```diff
@@ -1,43 +1,68 @@
 import json
 import math
+import uuid
 from contextlib import contextmanager
 from copy import deepcopy
-from unittest.mock import ANY
+from unittest.mock import ANY, patch
+
+from django.test import SimpleTestCase, TestCase, override_settings
 
-import uuid
-from django.test import SimpleTestCase, override_settings
 from nose.tools import nottest
-from unittest.mock import patch
-from corehq.apps.es.utils import check_task_progress
 
+from corehq.apps.app_manager.tests.app_factory import AppFactory
+from corehq.apps.domain.shortcuts import create_domain
+from corehq.apps.es.apps import app_adapter
+from corehq.apps.es.case_search import case_search_adapter
+from corehq.apps.es.cases import case_adapter
+from corehq.apps.es.domains import domain_adapter
+from corehq.apps.es.forms import form_adapter
+from corehq.apps.es.users import user_adapter
```

Comment:
It would be helpful if the tests for these adapters were moved to new test modules. Putting them in `test_client.py` is a bit confusing because the code being tested isn't from `client.py`. Adding new test modules (`test_apps.py`, `test_cases.py`, `test_users.py`, etc) has the added benefit of providing a home for future tests that are currently missing for those modules.

Reply:
I was actually tied between moving them to new files or to keep them in `test_client.py`  and my rationale for putting them in the client file was that we are testing a specific functionally of the client. 
But I see your suggestion makes more sense of having separate files for every adapter.